### PR TITLE
Remove flow logs from gateway page and some of TODOs

### PR DESCRIPTION
**Pull request by AndrewDryga**

### CODE REVIEW

File: elixir/apps/web/lib/web/live/resources/show.ex [hunk 1]
```diff
@@ -21,7 +21,6 @@ defmodule Web.Resources.Show do
           actor_groups_peek: Map.fetch!(actor_groups_peek, resource.id),
           flows: flows,
           params: Map.take(params, ["site_id"]),
-          todos_enabled?: Config.todos_enabled?(),
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/resources/show.ex:24]: Was unsed

File: elixir/apps/web/lib/web/live/gateways/show.ex [hunk 2]
```diff
@@ -62,10 +54,6 @@ defmodule Web.Gateways.Show do
             <:label>Name</:label>
             <:value><%= @gateway.name %></:value>
           </.vertical_table_row>
-          <.vertical_table_row :if={@todos_enabled?}>
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/gateways/show.ex:65]: This is irrelevant with our current design

**Review by jamilbk — COMMENTED** [review]
LGTM, will defer approval to @bmanifold

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**